tools-public/toolpad/components/DownloadsTable.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
+import { createComponent } from "@mui/toolpad/browser";
+import { DataGrid } from '@mui/x-data-grid';
+import { getPackages } from "./NpmChart";
+
+export interface DownloadsTableProps {
+  data: any[]
+}
+
+const getColumns = (packages) => ([
+  { field: 'date', headerName: 'Month', width: 150 },
+  ...packages.map(packageName => ({
+    field: packageName, headerName: packageName, width: packageName === '@radix-ui/react-primitive' || packageName === '@headlessui/react' ? 170 : 120
+  })),
+  {field: 'ratio', headerName: "Base UI marketshare", width: 150}
+]);
```

Comment:
```suggestion
const getColumns = (packages) => ([
  { field: 'date', headerName: 'Month', width: 150 },
  ...packages.map(packageName => ({
    field: packageName, headerName: packageName, width: 150
  })),
  {field: 'ratio', headerName: "Base UI marketshare", width: 150}
]);
```

cc @apedroferreira this is how you can reproduce the issue with the table overflowing.